lib/manageiq/messaging/kafka/client.rb

```diff
@@ -69,6 +69,11 @@ def topics
           Rdkafka::Metadata.new(native_kafka).topics.collect { |topic| topic[:topic_name] }
```

Comment:
@nasark I think we can use this method if we use:
```suggestion
          kafka_client.admin.metadata.topics.map { |topic| topic[:topic_name] }
```